perl/p5-mail-spamassassin/Portfile

```diff
@@ -4,8 +4,7 @@ PortSystem          1.0
 PortGroup           perl5 1.0
 
 perl5.branches      5.26 5.28 5.30
-perl5.setup         Mail-SpamAssassin 3.4.4
-revision            1
+perl5.setup         Mail-SpamAssassin 3.4.5
 categories-append   mail
```

Comment:
`revision` should still be present, just reset back to zero since the `version` is incrementing.

Reply:
Done. Also stripped trailing whitespace.